src/physics/World.js

```diff
@@ -1001,29 +1001,235 @@ Phaser.Plugin.Isometric.Arcade.prototype = {
      * If you need to calculate from the center of a display object instead use the method distanceBetweenCenters()
      *
      * @method Phaser.Plugin.Isometric.Arcade#distanceToXYZ
-     * @param {any} displayObject - The Display Object to test from.
+     * @param {any} displayObjectBody - The Display Object to test from.
      * @param {number} x - The x coordinate to test to.
      * @param {number} y - The y coordinate to test to.
      * @param {number} z - The y coordinate to test to
      * @return {number} The distance between the object and the x/y coordinates.
      */
-    distanceToXYZ: function (displayObject, x, y, z) {
+    distanceToXYZ: function (displayObjectBody, x, y, z) {
 
-        this._dx = displayObject.x - x;
-        this._dy = displayObject.y - y;
-        this._dz = displayObject.z - z;
+        this._dx = displayObjectBody.x - x;
+        this._dy = displayObjectBody.y - y;
+        this._dz = displayObjectBody.z - z;
 
         return Math.sqrt(this._dx * this._dx + this._dy * this._dy + this._dz * this._dz);
 
+    },
+
+    /**
+    * Find the distance between a display object (like a Sprite) and a Pointer. If no Pointer is given the Input.activePointer is used.
+    * The calculation is made from the display objects x/y coordinate. This may be the top-left if its anchor hasn't been changed.
+    * If you need to calculate from the center of a display object instead use the method distanceBetweenCenters()
+    * The distance to the Pointer is returned in isometric distance.
+    *
+    * @method Phaser.Physics.Arcade#distanceToPointer
+    * @param {any} displayObjectBody - The Display Object to test from.
+    * @param {Phaser.Pointer} [pointer] - The Phaser.Pointer to test to. If none is given then Input.activePointer is used.
+    * @return {number} The distance between the object and the Pointer.
+    */
+    distanceToPointer: function (displayObjectBody, pointer) {
+
+        pointer = pointer || this.game.input.activePointer;
+        var isoPointer = this.game.iso.unproject(pointer.position,undefined,displayObjectBody.z);
+        isoPointer.z = displayObjectBody.z;
+        var a = this.anglesToXYZ(displayObjectBody, isoPointer.x, isoPointer.y, isoPointer.z);
+
+        return a.r;
+
+    },
+
+    /**
+     * Find the angles in radians between a display object (like a IsoSprite) and the given x/y/z coordinate.
+     *
+     * @method Phaser.Physics.Isometric.Isometric.Arcade#anglesToXYZ
+     * @param {any} displayObjectBody - The Display Object to test from.
+     * @param {number} x - The x coordinate to get the angle to.
+     * @param {number} y - The y coordinate to get the angle to.
+     * @param {number} z - The z coordinate to get the angle to.
+     * @return {number} The angle in radians between displayObjectBody.x/y to Pointer.x/y
+     */
+    anglesToXYZ: function (displayObjectBody, x, y, z) {
+
+        // Spherical polar coordinates
```

Comment:
Image of spherical co-ords [here](https://en.wikipedia.org/wiki/Spherical_coordinate_system) (if you haven't come across them before).